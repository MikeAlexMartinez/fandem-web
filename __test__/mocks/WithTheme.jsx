import React from 'react';
import { ThemeProvider } from '@material-ui/styles';
import muiTheme from '../../styles/mui-theme';

const WithTheme = ({ children }) => (
  <ThemeProvider theme={muiTheme}>
    {children}
  </ThemeProvider>
);

export default WithTheme;
