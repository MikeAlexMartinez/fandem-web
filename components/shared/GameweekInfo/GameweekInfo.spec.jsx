import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import WithTheme from '../../../__test__/mocks/WithTheme';import GameweekInfoComponent from './index';

describe('<GameweekInfo />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<GameweekInfoComponent title="test title" toggleGameweek={() => {}} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
