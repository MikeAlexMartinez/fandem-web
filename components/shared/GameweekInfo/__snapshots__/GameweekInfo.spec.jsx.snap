// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameweekInfo /> renders and matches the snapshot 1`] = `
<GameweekInfo
  classes={
    Object {
      "example": "GameweekInfo-example-1",
    }
  }
  title="test title"
  toggleGameweek={[Function]}
/>
`;
